frontend/public/components/utils/name-value-editor.jsx

```diff
@@ -67,8 +67,8 @@ export const NameValueEditor = withDragDropContext(class NameValueEditor extends
     return <React.Fragment>
       <div className="row">
         {!readOnly && allowSorting && <div className="col-xs-1 co-empty__header"></div>}
-        <div className="col-xs-5 text-secondary text-uppercase">{nameString}</div>
-        <div className="col-xs-5 text-secondary text-uppercase">{valueString}</div>
+        <div className="col-xs-5 text-secondary text-uppercase co-full__header">{nameString}</div>
```

Comment:
I don't see where `co-full__header` is defined?

Reply:
I think that's leftover cruft from an earlier go at restyling. Removed it.